mw-webapp/src/logic/aboutProjectPage/TeamMember/teamMembers.ts

```diff
@@ -6,7 +6,7 @@ export const teamMembers: TeamMemberType[] = [
     id: 1,
     name: "Viktar Veratsennikau",
     description: "Founder",
-    imageUrl: "https://lh3.googleusercontent.com/drive-viewer/AKGpihawXxK1rm9vcGMcu2iUZ5RtxoIPAwgOvNuAtoOGcZKfoJ6O7yAaUTZ8k5sDwTNOxg4Lgcwch_nqlroh9t5uV94bNzIHjyjSsw=s1600-v0",
+    imageUrl: "",
```

Comment:
why did you remove my image? Don't you like it :)

Reply:
try to replace it on the `https://lh3.google.com/u/0/d/18oHI9KoiaYvd_UowHyqsJbDLLhmuxPxr=w2000-h2284-iv2` if it does not work for now

Reply:
I got an error about too many characters, the only way to avoid it was to delete this line. 
Replaced.

Reply:
Do you think it is a right solution?